lib/rules/links/reliability.js

```diff
@@ -0,0 +1,24 @@
+const self = {
+    name: 'links.reliability'
+,   section: 'document-body'
+};
+
+exports.name = self.name;
+const unreliableLinks = ['w3c-test.org'];
+
+exports.check = function (sr, done) {
+    sr.jsDocument.querySelectorAll("a").forEach(function (element) {
+        var href = element.href;
+        var splitLink = href.startsWith('http') && href.split('/');
+        var domain = splitLink && splitLink[2];
```

Comment:
It's best to use the [`url` module](https://nodejs.org/api/url.html) to get the hostname.

```suggestion
        const url = new URL(element.href);
        const domain = url.hostname;
```